### ISSUE TITLE
Support for swimming

### ISSUE THREAD

**Issue by MoonBarc** [cabaletta/baritone#1483]
## Describe your suggestion
Instead of walk-jumping super slowly through the water, using the 1.13 water sprinting mechanics would be a much better use of time and energy.

## Settings
`maxSwimDepth` - The depth of the water required to start swimming.
`breathInterval` - Every [x] seconds, the bot should come up to breath.
`safeBreathePaths` - Make sure that the bot doesn't through covered water, so that it can always breathe if it needs to.
`swimDepth` - I know it's a little close to `maxSwimDepth`, and is a little different, so maybe we would have to come up with a better name for it. But `swimDepth` would control how deep the player swims.

## Context
This would improve baritone by reducing the time needed when going to places with water nearby.

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by bokja** [comment]
I wish i knew how to fix this.

**Comment by ngoomie** [comment]
This would be really helpful, I always see Baritone travelling oh-so-slowly over large oceans and for it to be able to actually swim would probably shave massive amounts of time off of pathing for me

**Comment by Outersquid258** [comment]
Yeah this feature needs to be implemented, it would be greatly appreciated.

**Comment by ghost** [comment]
ts needs to get added sooner
